package.json

```diff
@@ -15,7 +15,8 @@
     "prebuild": "npm run rebuild && npm run browserify-preload",
     "browserify-preload": "browserify -o js/preload/_preloadMain.js -x electron --insert-global-vars=__filename,__dirname js/preload/preloadMain.js",
     "rebuild": "electron-rebuild -f",
-    "test": "npm run lint && jest --verbose --testPathPattern test --runInBand",
+    "test": "npm run lint && jest --verbose --testPathPattern test && npm run rebuild && npm run spectron-test",
+    "spectron-test": "jest --config tests/spectron/jest_spectron.json --runInBand && npm run rebuild",
```

Comment:
We should have a script that runs both the spectron and unit tests one after the other.